edx_solutions_api_integration/users/views.py

```diff
@@ -753,6 +753,18 @@ def post(self, request, user_id):  # pylint: disable=R0915
             existing_user.username = username
             response_data['username'] = existing_user.username
             existing_user.save()
+            if old_username != username:
```

Comment:
This code is repeating at line 769 too. Did you forget removing the old code?

Reply:
Done